spec/lib/redis/config_spec.rb

```diff
@@ -4,19 +4,22 @@
   context 'when single redis instance is used' do
     let(:redis_url) { 'redis://my-redis-instance:6379' }
     let(:redis_pasword) { 'some-strong-password' }
+    let(:redis_db_num) { '2' }
```

Comment:
let's take the case where Redis db number is specified as a separate case in the specs and ensure the default one remains 0 when environment variable is not present

Reply:
done